api/db-cdr/changelog-data/db.changelog-achilles_results13.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+  <changeSet author="biopete" id="changelog-13" runAlways="true" context="local or cloud">
+
+    <validCheckSum>ANY</validCheckSum>
+
+    <delete tableName="achilles_results"/>
+
+    <loadData tableName="achilles_results" file="csv/ACHILLES_results.csv" encoding="UTF-8" quotchar='"'>
+      <column name="analysis_id" type="NUMERIC"/>
+      <column name="stratum_1" type="STRING"/>
+      <column name="stratum_2" type="STRING"/>
+      <column name="stratum_3" type="STRING"/>
+      <column name="stratum_4" type="STRING"/>
+      <column name="stratum_5" type="STRING"/>
+      <column name="count_value" type="NUMERIC"/>
+    </loadData>
+
+  </changeSet>
+
+  <changeSet author="biopete" id="changelog-13.1 fill achilles_results_concept">
+    <sql
+            splitStatements="true"
+            stripComments="true">insert into achilles_results_concept
```

Comment:
Where is achilles_results_concept used below?

Reply:
I'm sorry, I'm asking where you actually use this table in the API. I want to make sure we're not including stuff that isn't used.

Or if there are cases where we're creating tables just as intermediate step toward creating other tables, we should know that (and potentially drop them afterwards.)